src/v2/Components/ArtworkFilter/index.tsx

```diff
@@ -309,6 +321,18 @@ export const BaseArtworkFilter: React.FC<
 
           <Spacer mb={2} />
 
+          {enableCreateAlert && artistId && artistName && (
+            <FiltersPills
+              artistId={artistId}
+              artistName={artistName}
+              aggregations={filterContext.aggregations!}
+              filters={filterContext.currentlySelectedFilters?.() || {}}
```

Comment:
**suggestion**: we don't have to pass `filters` as a separate prop. We can just use the `useArtworkFilterContext` hook in the component (in our case, `SavedSearchAlertForm`)